<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="1" ignored="0" total="8" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="ShredIt" duration-ms="73823" started-at="2017-08-08T20:34:32Z" finished-at="2017-08-08T20:35:46Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="73823" started-at="2017-08-08T20:34:32Z" finished-at="2017-08-08T20:35:46Z">
      <class name="eca.blogDetailsTest.BlogPageTest">
        <test-method status="PASS" signature="setup(java.lang.String, java.lang.String)[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="setup" is-config="true" duration-ms="6450" started-at="2017-08-08T16:34:32Z" finished-at="2017-08-08T16:34:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://shredit-qa.ecentricarts.com/en-us/home]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="beforeClass" is-config="true" duration-ms="3" started-at="2017-08-08T16:34:39Z" finished-at="2017-08-08T16:34:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="methodName(java.lang.reflect.Method)[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="methodName" is-config="true" duration-ms="0" started-at="2017-08-08T16:34:39Z" finished-at="2017-08-08T16:34:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void eca.blogDetailsTest.BlogPageTest.testBannerContents() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodName -->
        <test-method status="PASS" signature="testBannerContents()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testBannerContents" duration-ms="4" started-at="2017-08-08T16:34:39Z" finished-at="2017-08-08T16:34:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBannerContents -->
        <test-method status="PASS" signature="methodName(java.lang.reflect.Method)[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="methodName" is-config="true" duration-ms="0" started-at="2017-08-08T16:34:39Z" finished-at="2017-08-08T16:34:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void eca.blogDetailsTest.BlogPageTest.testContinueReadingLink() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodName -->
        <test-method status="FAIL" signature="testContinueReadingLink()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testContinueReadingLink" duration-ms="65914" started-at="2017-08-08T16:34:39Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testBannerContents" finished-at="2017-08-08T16:35:45Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 60 seconds waiting for visibility of element located by By.xpath: .//*[@id='page-content']/di/li[1]/p[3]/a
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'PPROPLESCH', ip: '192.168.100.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_141'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41), userDataDir=C:\Users\abharath\AppData\Local\Temp\scoped_dir15296_12560}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.115, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 8612944d1efa63ce06199e13a0cc0ac8]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 60 seconds waiting for visibility of element located by By.xpath: .//*[@id='page-content']/di/li[1]/p[3]/a
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'PPROPLESCH', ip: '192.168.100.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_141'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41), userDataDir=C:\Users\abharath\AppData\Local\Temp\scoped_dir15296_12560}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.115, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 8612944d1efa63ce06199e13a0cc0ac8
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at eca.utility.Waiting.waitForElementToLoad(Waiting.java:23)
	at eca.blogDetails.BlogPage.continueReadingLinkVerification(BlogPage.java:80)
	at eca.blogDetailsTest.BlogPageTest.testContinueReadingLink(BlogPageTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":".//*[@id='page-content']/di/li[1]/p[3]/a"}
  (Session info: chrome=59.0.3071.115)
  (Driver info: chromedriver=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 10.06 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'PPROPLESCH', ip: '192.168.100.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_141'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41), userDataDir=C:\Users\abharath\AppData\Local\Temp\scoped_dir15296_12560}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.115, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 8612944d1efa63ce06199e13a0cc0ac8
*** Element info: {Using=xpath, value=.//*[@id='page-content']/di/li[1]/p[3]/a}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContinueReadingLink -->
        <test-method status="SKIP" signature="testSubscribeBlog()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testSubscribeBlog" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testContinueReadingLink" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testSubscribeBlog()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testSubscribeBlog()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSubscribeBlog -->
        <test-method status="SKIP" signature="testArchivesContent()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testArchivesContent" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testEmailValidation" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testArchivesContent()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testArchivesContent()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testArchivesContent -->
        <test-method status="SKIP" signature="testEmailValidation()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testEmailValidation" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testSubscribeBlog" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testEmailValidation()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testEmailValidation()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmailValidation -->
        <test-method status="SKIP" signature="testBottomRailWidget()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testBottomRailWidget" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testEmailValidation" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testBottomRailWidget()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testBottomRailWidget()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testBottomRailWidget -->
        <test-method status="SKIP" signature="testOlderPostButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testOlderPostButton" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testEmailValidation" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testOlderPostButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testOlderPostButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOlderPostButton -->
        <test-method status="SKIP" signature="testRssButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="testRssButton" duration-ms="0" started-at="2017-08-08T16:35:45Z" depends-on-methods="eca.blogDetailsTest.BlogPageTest.testBottomRailWidget" finished-at="2017-08-08T16:35:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BlogPageTest.testRssButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BlogPageTest.testRssButton()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRssButton -->
        <test-method status="PASS" signature="close()[pri:0, instance:eca.blogDetailsTest.BlogPageTest@55b7a4e0]" name="close" is-config="true" duration-ms="1068" started-at="2017-08-08T16:35:45Z" finished-at="2017-08-08T16:35:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- eca.blogDetailsTest.BlogPageTest -->
    </test> <!-- Test -->
  </suite> <!-- ShredIt -->
</testng-results>
